opcua.cpp

```diff
@@ -1611,8 +1816,19 @@ void OPCUA::browse(const string &nodeid, vector<string> &variables)
 	{
 		if (browseResult.references[i].nodeClass == OpcUa_NodeClass_Object)
 		{
+			bool processNode = checkFiltering(browseResult.references[i].browseName, browseResult.references[i].nodeClass);
+			if(!processNode)
+			{
+				Logger::getLogger()->info("%s:%d: Skipping Browse Node '%s' with browseName '%s', because of filtering config",
```

Comment:
This seems reasonable as an _Info_ message (rather than _Debug_), but remove the _FunctionName_ and the _Line_ number. System administrators do not care what we named our methods or what line was executing.